packages/manager/modules/nasha/src/components/partition/delete/delete.controller.js

```diff
@@ -15,4 +18,9 @@ export default class NashaComponentsPartitionDeleteController {
       )
       .catch((error) => this.close({ error }));
   }
+
+  onClickOnClose() {
```

Comment:
Same as https://github.com/ovh/manager/pull/7762/files#r943141172

Reply:
It's applied.